### ISSUE TITLE
Can't load "Real-time statistics" on the tribler website

### ISSUE THREAD

**Issue by Kcchouette** [Tribler/tribler.github.com#27]
1. going on https://tribler.org/tunnel-stats.html using latest Firefox.
2. See nothing
3. see that in the console, there are errors.

Errors are ```Blocked loading mixed active content “http://95.211.198.143:8667/history?callback=xxx”```
It comes from: [tunnel-stats.html, L.70](https://github.com/Tribler/tribler.github.com/blob/768d9831238199bf988351b8c7cd824bc137eef1/tunnel-stats.html#L70)
This error happened because I am on the https version of the website, and the url from tunnel-stats is http.
And starting in Firefox 23, mixed active content is blocked by default, see https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content for more information

**Comment by qstokkink** [comment]
Related to Tribler/tribler#1567 .

**Comment by Kcchouette** [comment]
why have you close the issue, @qstokkink .
The problem is still occuring...

**Comment by qstokkink** [comment]
@Kcchouette oh, sorry, I closed a whole group of the statistics issues at once: the tunnel-stats.html has been superseded by http://statistics.tribler.org/ . We now properly have a 404 on the old page as well.
